extensions/pyRevitTools.extension/checks/cad_audit_check.py

```diff
@@ -106,32 +106,38 @@ def check_model(doc, output):
     
     table_data = [] # store array for table formatted output
     row_head = ["No", "Select/Zoom", "DWG instance", "Loaded status", "Workplane or single view", "Duplicate", "Workset", "Creator user", "Location site name"] # output table first and last row
+    row_empty = ["", "", "No CAD instances found", "", "", "", "", "", ""] # output table row for when no CAD found
+    cad_inst_count = 0
     cad_instances = collect_cadinstances(coll_mode)
-    for count, cad in enumerate(cad_instances, start=1):
-        cad_id = cad.Id
-        cad_is_link = cad.IsLinked
-        cad_name = cad.Parameter[DB.BuiltInParameter.IMPORT_SYMBOL_NAME].AsString()
-
-        table_row = [
-            count,
-            output.linkify(cad_id, title="Select"),
-            cad_name,
-            get_load_stat(cad, cad.IsLinked), # loaded status
-        ]
-
-        # if the instance has an owner view, it was placed on the active view only (bad, so give warning and show the view name)
-        # if the instance has no owner view, it should have a level or workplane (good)
-        cad_own_view_id = cad.OwnerViewId
-        if cad_own_view_id == DB.ElementId.InvalidElementId:
-            table_row.append(doc.GetElement(cad.LevelId).Name)
-        else:
-            cad_own_view_name = doc.GetElement(cad_own_view_id).Name
-            table_row.append(":warning: view '{}'".format(cad_own_view_name))
-        table_row.append(":warning:" if cad_name in [row[2] for row in table_data] else "-") # If the name is already in table_data, it is a duplicat (bad)
-        table_row.append(revit.query.get_element_workset(cad).Name) # cad instance workset
-        table_row.append(DB.WorksharingUtils.GetWorksharingTooltipInfo(revit.doc, cad.Id).Creator) # ID of the user
-        table_row.append(get_cad_site(cad)) # Extract site name from location
-        table_data.append(table_row)
+    if cad_instances:
```

Comment:
invert the if condition, so that the shorter branch (the append of the empty row) is easier to track

Reply:
If you did, you forgot to push the changes to your repo... 😉